plugin-places/src/main/java/com/mapbox/mapboxsdk/plugins/places/picker/ui/PlacePickerActivity.java

```diff
@@ -207,6 +289,31 @@ public void onClick(View view) {
     });
   }
 
+  /**
+   * Bind the device location Floating Action Button to this activity's UI and move the
+   * map camera if the button's clicked.
+   */
+  private void addUserLocationButton() {
+    userLocationButton.show();
+    userLocationButton.setOnClickListener(new View.OnClickListener() {
+      @Override
+      public void onClick(View view) {
+        if (mapboxMap.getLocationComponent().getLastKnownLocation() != null) {
```

Comment:
You can assign last know location to a local field and re-use that across this method.

Reply:
Fixed